### PR TITLE
Feat/tests

### PR DESCRIPTION
**Pull request by bazimov**

### CODE REVIEW

File: src/lavatory/commands/policies.py [hunk 1]
```diff
@@ -32,12 +32,8 @@
...
     """Prints out a JSON list of all repos and policy descriptions."""
     LOG.debug('Passed args: %s, %s, %s, %s', ctx, policies_path, repo, repo_type)
-    artifactory = Artifactory(repo_name=None)
-    all_repos = artifactory.list(repo_type=repo_type)
-    if repo:
-        selected_repos = repo
-    else:
-        selected_repos = all_repos.keys()
+
+    storage_info, selected_repos = get_artifactory_info(repo_name=None, repo_type=repo_type)
```
Comment (ndcampbell) [inline src/lavatory/commands/policies.py]: If this only ever passes `repo_name = None`, how does it get repos specified in the CLI?

File: src/lavatory/commands/purge.py [hunk 2]
```diff
@@ -36,15 +37,11 @@
...
     """Deletes artifacts based on retention policies."""
     LOG.debug('Passed args: %s, %s, %s, %s, %s, %s', ctx, dryrun, policies_path, default, repo, repo_type)
-    artifactory = Artifactory(repo_name=None)
-    before_purge_data = artifactory.list(repo_type=repo_type)
-    if repo:
-        selected_repos = repo
-    else:
-        selected_repos = before_purge_data.keys()
+
+    storage_info, selected_repos = get_artifactory_info(repo_name=None, repo_type=repo_type)
```
Comment (ndcampbell) [inline src/lavatory/commands/purge.py]: If this only ever passes `repo_name = None`, how does it get repos specified in the CLI?

File: src/lavatory/commands/purge.py [hunk 3]
```diff
@@ -36,15 +37,11 @@
 def purge(ctx, dryrun, policies_path, default, repo, repo_type):
     """Deletes artifacts based on retention policies."""
     LOG.debug('Passed args: %s, %s, %s, %s, %s, %s', ctx, dryrun, policies_path, default, repo, repo_type)
-    artifactory = Artifactory(repo_name=None)
-    before_purge_data = artifactory.list(repo_type=repo_type)
```
Comment (ndcampbell) [inline src/lavatory/commands/purge.py]: This variable gets used on line 86. Was this able to even run?

File: src/lavatory/utils/get_artifactory_info.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+import logging
+
+from .artifactory import Artifactory
+
+
+def get_artifactory_info(repo_name=None, repo_type='local'):
+    """Get storage info from Artifactory.
+
+    Args:
+        repo_name (tuple, optional): Name of artifactory repo.
```
Comment (ndcampbell) [inline src/lavatory/utils/get_artifactory_info.py]: `repo_names` might be better if this is expected to be a tuple with multiple values.

File: src/lavatory/commands/purge.py (hunk 3 above)
Comment (bazimov) [reply src/lavatory/commands/purge.py]: Looks like in line 86, it is passed as argument which I replaced in line 44.

File: src/lavatory/utils/get_artifactory_info.py (hunk 4 above)
Comment (bazimov) [reply src/lavatory/utils/get_artifactory_info.py]: Changed to plural.

File: src/lavatory/commands/purge.py (hunk 3 above)
Comment (ndcampbell) [reply src/lavatory/commands/purge.py]: ahhh ok, my bad

Comment (ndcampbell) [reply src/lavatory/commands/purge.py]: ah ok, my bad

**Pull request merged**